src/icons/Thread.tsx

```diff
@@ -0,0 +1,26 @@
+export default function () {
+  return (
+    <svg
+      width="24"
+      height="24"
+      viewBox="0 0 24 24"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
+        d="M16 9V8C16 6.89543 15.1046 6 14 6H4C2.89543 6 2 6.89543 2 8V14.7059C2 15.8105 2.89543 16.7059 4 16.7059H8.20685C8.71631 16.7059 9.20658 16.9003 9.57758 17.2495L11.1293 18.7099C11.8994 19.4347 13.1006 19.4347 13.8707 18.7099L15.3707 17.2981C15.7723 16.9202 16 16.3932 16 15.8417V15"
+        stroke="#EFECD6"
+        stroke-width="2"
+        stroke-linecap="round"
```

Comment:
Let's remove `stroke-linecap="round"` and `fill="none"` and `        stroke-linejoin="round"`